src/acc.rs

```diff
@@ -100,14 +191,60 @@ impl Paintable {
                     *v = norm(v)?;
                 }
                 Paintable::Mech(m.clone(), ps)
+            },
+            Paintable::NonUniformMech(m, ps, ns) => {
+                let mut ps = ps.clone();
```

Comment:
So that would become

```rust
let ps = ps.iter().map(|(k, v)| norm(v).map(|v| (k.to_string(), v))).collect::<Result<Map<String, String>>>()?;
```

which is not that much clearer imho

Reply:
https://github.com/thorstenhater/nmlcc/pull/23/commits/2ff328f8960fa33118d1ab0c4513d89605ce0169